python/redmonster/application/zfind.py

```diff
@@ -28,7 +28,7 @@
 from redmonster.physics import misc
 from time import gmtime, strftime
 from os.path import exists
-from ConfigParser import SafeConfigParser
+from configparser import SafeConfigParser
```

Comment:
I think our DESI conda installs at NERSC have the separate `configparser` backport package installed, so that this will still work on Python 2.7.  However, this could break in a Python 2.7 environment that did not have that installed.